first_nes.s

```diff
@@ -73,6 +73,9 @@
 .SEGMENT "SPRITES"
 .INCLUDE "data/sprites/small_luigi.inc"
 
+; Background nametable and attribute data
+.INCLUDE "data/background/scene.inc"
+
```

Comment:
Referencing https://github.com/gregkrsak/first_nes/blob/master/config/ines.cfg#L90 here...

Should we create a new segment identifier (vs. using SPRITES) for the background?

Reply:
youre right this should be done